README.md

```diff
@@ -179,7 +179,7 @@ module "landing_zone" {
 | aws\_deny\_leaving\_org | Enable SCP that denies accounts the ability to leave the AWS organisation | `bool` | `true` | no |
 | aws\_deny\_root\_user\_ous | List of AWS Organisation OUs to apply the "DenyRootUser" SCP to | `list(string)` | `[]` | no |
 | aws\_guardduty | Whether AWS GuardDuty should be enabled | `bool` | `true` | no |
-| aws\_okta\_group\_ids | List of Okta group IDs that should be assigned the AWS SSO Okta app | `list(string)` | `[]` | no |
+| aws\_okta\_group\_ids | List of Okta group IDs that should be assigned to the AWS SSO Okta app | `list(string)` | `[]` | no |
```

Comment:
This should be reverted as it doesn't make sense 🙂

Reply:
agree